firestore-shorten-urls-bitly/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.2
+
+fixed - Migrates Bitly API to v4 since v3 will be deactivated on March 1, 2020 (Issue #170).
+
+**Important:** Update your extension to at minimum v0.1.2 before March 1, 2020; otherwise, your installed extension will stop working.
```

Comment:
**Important:** Update your extension to at minimum v0.1.2 before March 1, 2020; otherwise, your installed extension will stop working. <a href="https://dev.bitly.com/deprecated.html">Learn more</a>.

Reply:
added learn more link.